life_dashboard/dashboard/forms.py

```
@@ -1,15 +1,91 @@
+            user.email = self.cleaned_data["email"]
+            user.save()
+            profile.save()
+        return profile
+
+
+class ProfileUpdateForm(forms.ModelForm):
+    first_name = forms.CharField(max_length=100, required=False)
+    last_name = forms.CharField(max_length=100, required=False)
+    email = forms.EmailField(required=False)
+
+    class Meta:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)